yarn.lock

```diff
@@ -12277,7 +12277,7 @@ joycon@^2.2.5:
   resolved "https://registry.yarnpkg.com/joycon/-/joycon-2.2.5.tgz#8d4cf4cbb2544d7b7583c216fcdfec19f6be1615"
   integrity sha512-YqvUxoOcVPnCp0VU1/56f+iKSdvIRJYPznH22BdXV3xMk75SFXhWeJkZ8C9XxUWt1b5x2X1SxuFygW1U0FmkEQ==
 
-jpeg-js@0.2.0, jpeg-js@^0.4.0:
+jpeg-js@0.2.0, jpeg-js@^0.4.3:
```

Comment:
This change was automatic from running yarn install for updated package.json.

Reply:
This was merged into `master` earlier today. I went ahead and merged `master` into your branch

Reply:
Coolio!  Thx, Theo!  :)